layout/_third-party/katex.swig

```diff
@@ -0,0 +1,6 @@
+{% if theme.katex.enable %}
+  {% if not theme.katex.per_page or (page.total or page.mathjax) %}
+    <link rel="stylesheet" href="{{ theme.katex.cdn }}" type="text/css">
```

Comment:
According to [Server side rendering or rendering to a string](https://github.com/Khan/KaTeX#server-side-rendering-or-rendering-to-a-string), it is no need to add the js file.

Only the CSS file is enough.